app/views/admin/external_reports/_form.html.haml

```diff
@@ -23,4 +23,7 @@
         %li
           Report Type:
           = f.select :report_type, report.options_for_report_type, {}, prompt: 'report type:'
+        %li
+          Allowed for Students:
+          = f.check_box :allowed_for_students
```

Comment:
We should add some extra description here like:
"If the report type is 'offering', then students will see a button to open the report"

Reply:
Done.